salesforce/10.1/modules/ROOT/pages/salesforce-connector-reference.adoc

```diff
@@ -3971,6 +3971,11 @@ Subscribe to a topic. First, the topic must be published and after that, a subsc
 ** GB | The unit in which maxInMemorySize is expressed |  |
 |===
 
+[non-repeatable-stream]
+=== Non-repeatable Stream
+
+Although uncommon, there are cases in which you might want to disable the repeatable stream functionality and use non repeatable streams, for example, in cases where you don't want to pay for extra memory or have performance overhead.
```

Comment:
Might be worth a slight rewrite? "In a few rare cases, you may wish to disable the repeatable stream functionality and use non repeatable streams, which can have less performance overhead, less memory use, and less cost."